roles/kubernetes/node/templates/kubelet.kubeadm.env.j2

```diff
@@ -48,6 +48,9 @@ KUBELET_HOSTNAME="--hostname-override={{ kube_override_hostname }}"
 {% else %}
 --fail-swap-on={{ kubelet_fail_swap_on|default(true)}} \
 {% endif %}
+{% if dynamic_kubelet_configuration %}
+--dynamic-config-dir={{ kube_config_dir }}/dynamic_kubelet_dir \
```

Comment:
Could we put this path in the defaults file? Else you have to look in the template to find it?

Reply:
There is no default directory, you have to define it, then kubelet create it.

Reply:
> There is no default directory, you have to define it, then kubelet will create it.

Yeah. I know. But wouldn't it be smart to have it in defaults still ?

Reply:
1. I thought about it, I think the dynamic kubelet configuration should be consistent with the systemd kubelet env file.
2.And need to be compatible with more operating systems，It is difficult to control the write permissions of different systems.